ts/dx.all.d.ts

```diff
@@ -31421,7 +31422,8 @@ declare module DevExpress.viz {
     export type OptionChangedEvent = DevExpress.events.EventInfo<dxPieChart> &
       DevExpress.events.ChangedOptionInfo;
     export type PieChartAnnotationLocation = 'center' | 'edge';
-    export type PieChartLabelOverlap = 'hide' | 'none' | 'shift';
+    export type PieChartLabelOverlap =
+      DevExpress.common.charts.ShiftLabelOverlap;
```

Comment:
After apply of the previous suggestions we should add
`
declare module DevExpress.common.charts {
  /** @deprecated Use DevExpress.common.charts.ShiftLabelOverlap instead */
  export type FunnelLabelOverlap = DevExpress.common.charts.ShiftLabelOverlap;
  /** @deprecated Use DevExpress.common.charts.ShiftLabelOverlap instead */
  export type PieChartLabelOverlap = DevExpress.common.charts.ShiftLabelOverlap;
}
`
to `/ts/aliases.d.ts` file

Reply:
done